### PR TITLE
fixed add-user and remove-user APIs

### PR DESCRIPTION
**Pull request by KUSHAL-OMER**
Now, we can remove a user from a group and add a new one. Both the effects will be reflected in the Groups and Users Database collections.

### CODE REVIEW

File: controllers/groupController.js [hunk 1]
```diff
@@ -2,26 +2,31 @@ const opencage = require("opencage-api-client");
 const Message = require("../models/messageModel");
 const Group = require("../models/groupModel");
 const User = require('../models/userModel');
-
-const opencage = require('opencage-api-client');
```
Comment (BharatShukla30) [inline controllers/groupController.js:6]: why did you remove this??

File: routes/groupRoute.js [hunk 2]
```diff
@@ -6,8 +6,6 @@ const router = express.Router();
 const { createGroup, addUser, makeGroupPermanent, removeUser, nearestGroup, fetchLastMessages, fetchGroupDetails } = require("../controllers/groupController");
 
 
-const router = express.Router();
```
Comment (BharatShukla30) [inline routes/groupRoute.js:9]: is this not needed? why removing this?

**Review by BharatShukla30 — COMMENTED** [review]
PLease address this

File: controllers/groupController.js (hunk 1 above)
Comment (KUSHAL-OMER) [reply controllers/groupController.js:6]: All this was present already. I removed it because this line is already in the file.

File: routes/groupRoute.js (hunk 2 above)
Comment (KUSHAL-OMER) [reply routes/groupRoute.js:9]: All this was present already. I removed it because this line is already in the file.

**Review by BharatShukla30 — APPROVED** [review]
